tests/PackageFuncCall/PackageFuncCall.log

```diff
@@ -1474,7 +1473,6 @@ design: (work@top)
               \_io_decl: (sbox4), parent:prim_cipher_pkg::sbox4_64bit
                 |vpiName:sbox4
                 |vpiDirection:1
-                |vpiSize:121
```

Comment:
Out of curisoity: what are these nodes that had a vpiSize before but not anymore ?

Reply:
Bad casting, memory corruption